web/src/containers/TopBtns.js

```diff
@@ -5,13 +5,12 @@ import { connect } from 'react-redux';
 import { push } from 'react-router-redux';
 
 import { t } from '../i18n';
-import { logout } from '../actions/auth';
 import Btn from '../components/Btn';
 
 class TopBtns extends Component {
   handleLogout = e => {
     e.preventDefault();
-    this.props.logout();
+    this.props.pushRoute('/logout');
```

Comment:
Update the top nav button to route to `/logout` instead of calling the logout action directly